src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheck.java

```diff
@@ -33,27 +33,27 @@
  * </p>
  * <ul>
  * <li>
- * Property {@code applyToPackage} - Controls whether to apply the check to package-private member.
+ * Property {@code applyToPackage} - Controls whether to apply the check to package-private members.
```

Comment:
```suggestion
 * Property {@code applyToPackage} - Control if check should apply to package-private members.
```

Please do this for all.

Reply:
done